ext/user_request.stub.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/** @generate-class-entries */
```

Comment:
We've had users complain about us having multiple stub files and it thus making it easy to miss that there are others - in the PHP ecosystem stub files are generally a single file per extension, and I was looking into merging them eventually.

Could you please put it into ddtrace.stub.php? Thanks!

Reply:
I mean I can do it, but then I can't really include ddtrace_arginfo.h in user_request.c, otherwise I'll get a bunch of warning about all those unused statics + likely code bloat. Then if I don't I'll have global (non-static) function declarations for the PHP functions without prototypes, which is also a compiler warning (depending on the compiler and settings). Generally, it just seems like a less cohesive solution.

Are you sure about this?

Reply:
You don't need a prototype to define a function? You need a prototype to use it. ddtrace.c should not care of whether it's defined there or not, that's the linkers problem. So you don't need to include ddtrace_arginfo.h at all.

Reply:
what I mean is that gcc/clang will warn, depending on the settings:

```
glopes /tmp $ cat /tmp/a.c
int v() { return 2; }
int main() { return v(); }
glopes /tmp $ gcc  -Wmissing-prototypes /tmp/a.c
/tmp/a.c:1:5: warning: no previous prototype for ‘v’ [-Wmissing-prototypes]
    1 | int v() { return 2; }
      |     ^
glopes /tmp $ clang  -Wmissing-prototypes /tmp/a.c
/tmp/a.c:1:5: warning: no previous prototype for function 'v' [-Wmissing-prototypes]
int v() { return 2; }
    ^
/tmp/a.c:1:1: note: declare 'static' if the function is not intended to be used outside of this translation unit
int v() { return 2; }
^
static 
1 warning generated.
```

Reply:
Do we have it active? If not, it's stupid, but trivial to write `ZEND_FUNCTION(foo); ZEND_FUNCTION(foo) { ... }`.